apps/builder/app/shared/remix/handle-request.server.tsx

```diff
@@ -1,7 +1,14 @@
 import { renderToString } from "react-dom/server";
 import { RemixServer } from "@remix-run/react";
 import type { EntryContext } from "@remix-run/node";
-import { insertCriticalCss } from "./insert-critical-css";
+import { flushCss } from "@webstudio-is/design-system";
+import { renderHeadToString } from "remix-island";
+import { Head } from "./root";
+// eslint-disable-next-line import/no-internal-modules
+import interFont from "@fontsource/inter/variable.css";
+// eslint-disable-next-line import/no-internal-modules
+import manropeVariableFont from "@fontsource/manrope/variable.css";
```

Comment:
bad part about this structure is that now you have a central place for all fonts, before that builder, dashboard and login was managing their fonts

e.g. dashboard and login uses brand-defined stuff, where builder doesn't

Reply:
The good part that no fouc caused by remix-island

Reply:
Also I wrote a comment
https://github.com/webstudio-is/webstudio-builder/pull/1621/files#diff-2f32780da9f3d635a9c0b765d6cb7888c66e2db35f0572f3ea0971751a0814c1R28-R30